### PR TITLE
Weather logic

### PR DESCRIPTION
**Pull request by Tripple-A**
This pull request is from the weather logic branch which contains webpack and its dependencies needed for the weather app project. Async functions are deployed and the giffy and weathermap API are used.

### CODE REVIEW

**Comment by Tripple-A** [comment]
## Chrome refuses to load background image but others do,hence background image is different on Chrome than it is on other browsers. Webpack is used for computing everything into main.js and modules are not imported or exported as it was not stated as a requirement. All functions are on index.js but the css file is imported.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.20.0",
+    "file-loader": "^5.0.2",
+    "style-loader": "^1.1.3",
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10"
+  },
+  "dependencies": {
+    "giphy-api": "^2.0.1",
+    "webpack-dev-server": "^3.10.1"
```
Comment (adriaanbd) [inline package.json]: This should be in `devDependencies`, as specified [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) [`REQUIRED`]

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": "off",
+      "import/no-unresolved": "off"
```
Comment (adriaanbd) [inline .eslintrc.json]: Please remove these two statements and stick to the required eslint cofiguration:

```
{
  "env": {
    "browser": true,
    "es6": true,
    "jest": true
  },
  "parserOptions": {
    "ecmaVersion": 2018,
    "sourceType": "module"
  },
  "extends": ["airbnb-base"],
  "rules": {
    "no-shadow": "off",
    "no-param-reassign": "off",
    "eol-last": "off"
  }
}
``

[`REQUIRED`]

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    width: 80%;
+    float: left;
+}
+
+.content{
+    height:1000px;
+    background-attachment: fixed;
+     background-image: url('sky.jpg');
+    background-repeat: no-repeat;
+    background-position: center;
```
Comment (adriaanbd) [inline src/index.css]: Please fix indentation here [`REQUIRED`]

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+});
+
+btn2.addEventListener('click', () => {
+  if (deg.textContent === 'C') {
+    temp.textContent = farenheit(temp.textContent);
+    deg.textContent = 'F';
+  }
+});
+
+btn3.addEventListener('click', () => {
```
Comment (adriaanbd) [inline src/index.js]: Please make different requests for either F or C, as specified [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/requests_requirement.png). At the moment, the app is fetching the temperature in C and converting it to F. User should be able to click on F as fetch the temperature on F.

[`REQUIRED`]

**Comment by Tripple-A** [comment]
## All changes have been effected, except that of the eslint as it can be seen through careful observation that ESlint does not give any errors even when those comments are removed but stickler has an unprecedented issue with 'importing' while using 'require, hence it is necessary to turn it off. I have contacted those in charge of the Javascript linting files and I have been assured that the line  ' "import/no-unresolved": "off"' shall be added to the file to avoid further issues. Thank you

**Pull request merged**